flow360/component/simulation/outputs/output_entities.py

```diff
@@ -0,0 +1,35 @@
+from typing import List, Literal, Tuple, final
+
+import pydantic as pd
+
+from flow360.component.simulation.framework.entity_base import EntityBase, EntityList
+from flow360.component.simulation.primitives import Surface
+
+
+@final
+class Slice(EntityBase):
+    _entity_type: Literal["SliceType"] = "SliceType"
+    slice_normal: Tuple[float, float, float] = pd.Field()
+    slice_origin: Tuple[float, float, float] = pd.Field()
+
+
+@final
+class Isosurface(EntityBase):
```

Comment:
`Isosurface` as well as `Slice` are indeed geometrical representation (they are composed of triangles with connectivity) but just not the ones that exist in the original mesh.

They do not have to inherit from `EntityBase` though since they will not get registered by anyone. I removed the inheritance. Not sure if this is what you mean.

Reply:
Removed inheritance.